setup.py

```diff
@@ -23,23 +23,14 @@ def find_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 
-requires = ['jmespath>=0.7.1,<1.0.0',
-            'python-dateutil>=2.1,<2.7.0',
-            'docutils>=0.10']
-
-
-if sys.version_info[:2] == (2, 6):
-    # For python2.6 we have a few other dependencies.
-    # First we need an ordered dictionary so we use the
-    # 2.6 backport.
-    requires.append('ordereddict==1.1')
-    # Then we need simplejson.  This is because we need
-    # a json version that allows us to specify we want to
-    # use an ordereddict instead of a normal dict for the
-    # JSON objects.  The 2.7 json module has this.  For 2.6
-    # we need simplejson.
-    requires.append('simplejson==3.3.0')
-
+# The python_version specifiers are necessary for supporting older
+# versions of python.
+requires = 'docutils>=0.10'
```

Comment:
This needs to be an actual list, right now it's missing the `[` and `]`.

Also the first element needs a comma after it. if you added the square brackets right now, the first two entries would concatenate.

Reply:
ya, hand jamming doesn't always work the best (especially when distracted).